cmd/snap-bootstrap/initramfs_systemd_mount_test.go

```diff
@@ -55,6 +55,44 @@ exit 1
 	c.Assert(err, ErrorMatches, "mocked error")
 }
 
+// This is needed to properly split options that contain a ',' character that
+// is double escaped before passed as an argument to systemd-mount.
+func splitOptions(s string) []string {
+	var o []string
+
+	s = strings.ReplaceAll(s, "\\\\", "\\")
```

Comment:
Ditto on backtics and a comment that clarifies what is going on.

Reply:
I removed this call as it was not needed anymore. See previous comment.